### PR TITLE
V0.9.0

### PR DESCRIPTION
**Pull request by MrChico**
Draft PR just for visibility

### CODE REVIEW

**Comment by MrChico** [comment]
@denis-bogdanas This now passes the v0.9.0 tests and is ready for review!

**Comment by denis-bogdanas** [comment]
@MrChico I see it's linked to tests commit for v0.8.3.

**Comment by denis-bogdanas** [comment]
nevermind it's v0.9

File: beacon-chain.k [hunk 1]
```diff
@@ -334,7 +377,7 @@ def compute_activation_exit_epoch(epoch: Epoch) -> Epoch:
     return Epoch(epoch + 1 + ACTIVATION_EXIT_DELAY)
 */
   syntax Int ::= "compute_activation_exit_epoch" "(" Int ")" [function]
-  rule compute_activation_exit_epoch(EP) => EP +Int 1 +Int ACTIVATION_EXIT_DELAY
```
Comment (denis-bogdanas) [inline beacon-chain.k]: Can you also update python code in the comment, here and in other places?

Comment (MrChico) [reply beacon-chain.k]: I updated the comment here. I couldn't find any other places I missed though, do you still see any?

Comment (denis-bogdanas) [reply beacon-chain.k]: Thanks! I didn't see any other places missing. Just wrote a general message. Will review soon.

**Pull request merged**